helm/alfresco-content-services/values.yaml

```diff
@@ -518,6 +518,8 @@ alfresco-ai-transformer:
   # -- toggle deploying Alfresco ai transformer for more details about configuration check
   # https://github.com/Alfresco/alfresco-helm-charts/tree/main/charts/alfresco-ai-transformer
   enabled: false
+  # -- Enforce static resource naming in AIS so the ATS trouter can be given the URL of the service
+  fullnameOverride: alfresco-intelligence-service
```

Comment:
not sure why we need this when elsewhere there is plenty of components that needs to know other components URL - just reuse the named template which generate the correct/same service name?

Reply:
I used the `fullnameOverride` because the service name needs is passed as a generic  `.Values.environment.JAVA_OPTS` in the ATS chart. so I can't use templates, unless we change ATS chart to evaluate `.Values.environment.JAVA_OPTS` with tpl()